NoSoft-SA/nspack
# Feature/titan api schema

Summary of this change:
Added schema validation to the outgoing inspection request of the TITAN API.
As well as fixed a bug where the custom country name, instead of the ISO code, was sent through to TITAN.

How to test this code via the UI:
Run TITAN mock API and request inspection for any given consignment that already had a previous request sent.
Compare the JSON requests before and after this update to ensure they are still the same.
